srcCpp/RTIDDSImpl.cxx

```diff
@@ -2378,18 +2362,18 @@ IMessagingWriter *RTIDDSImpl<T>::CreateWriter(const char *topic_name)
         return NULL;
     }
 
-    if (strcmp(topic_name, perftest_cpp::_ThroughputTopicName) == 0) {
+    if (strcmp(topic_name,THROUGHPUT_TOPIC_NAME) == 0) {
```

Comment:
add extra space
```
if (strcmp(topic_name, THROUGHPUT_TOPIC_NAME) == 0) {
```

Do the dame for other languages and topics names

Reply:
Done.